docs/agent-policy.md

```diff
@@ -0,0 +1,185 @@
+# Elastic Agent Policies
+
+The policy of an Agent is the user-controlled portion of the Agent configuration that, together with [component spec files](component-specs.md), determines the components that Agent runs and their configuration.
+
+A __component__ is an executable that Agent runs or monitors. That component is divided into __units__ corresponding to its functional behavior. For example, Filebeat is a component, but Filebeat may be run with multiple `filestream` inputs, and each of those is a unit with its own configuration.
```

Comment:
```suggestion
A __component__ is an executable that Agent runs and monitors. That component is divided into __units__ corresponding to its functional behavior. For example, Filebeat is a component, but Filebeat may be run with multiple `filestream` inputs, and each of those is a unit with its own configuration.
```

Reply:
Elastic Agent always monitors it, not really an or.

Reply:
It always monitors but as I understand it it doesn't always run, so "runs and monitors" seemed wrong to me since it doesn't fit service components like endpoint. Maybe "runs and/or monitors"?